front/src/modules/timesStops/hooks/useTimesStopsColumns.tsx

```diff
@@ -37,41 +37,55 @@ export const useTimesStopsColumns = <T extends TimesStopsRow>(
   tableType: TableType,
   allWaypoints: T[] = []
 ) => {
-  const { t } = useTranslation('timesStops');
+  const { t } = useTranslation();
```

Comment:
To avoid `timeStopTable.` everywhere
```suggestion
  const { t } = useTranslation('', { keyPrefix: 'timeStopTable' });
```

Reply:
Done